index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portifolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <nav>
+                <ul>
+                    <li><a class="myLogo font-style" href="">Kenbon</a> </li>
+                    <li><a class="toggler-icon"><img src="images/humburger.png" alt="humberger-icon"></a>
+                    </li>
```

Comment:
- [ ] Great job adding the hamburger menu 💯, but I think it could be better if you wrap the hamburger icon in a `<nav>` tag, the `nav` tag is for semantics reasons 🥇. Something like this: 
```
<nav>
   <li>
      <a class="toggler-icon"><img src="images/humburger.png" alt="humberger-icon"></a> 
   </li>
</nav>
```